src/middleware/packages/void/service.js

```diff
@@ -263,18 +279,40 @@ module.exports = {
 
         // TODO : fetch the void file from remote servers instead ?
 
-        for (const server of Object.keys(serversMap)) {
+        for (const serverUrl of Object.keys(serversMap)) {
+
+          const voidUrl = urlJoin(serverUrl, '/.well-known/void');
+
+          let originalVoid;
+          try {
+            const response = await fetch(voidUrl, {
+              method: 'GET',
+              headers: {
+                Accept: 'application/ld+json'
+              }
+            });
+            if (response.ok) {
+              const json = await response.json();
+
+              let mapServers = {};
+              for (let s of json['@graph']) { mapServers[s['@id']] = s; }
+              const server = mapServers[createFragmentURL('', serverUrl)];
+              originalVoid = server;
+            }
+          } catch (e) { }
```

Comment:
Lorsqu'on ignore des erreurs, je trouve que c'est une bonne idée de quand même l'afficher dans le logger (`this.logger.warn("Siliently ignored error: " + e.message)`). Ca facilite le debugging ! Surtout dans le cas où l'erreur est due à une erreur de typo ou autre chose, sans avoir rien à voir avec une erreur de fetch.

Reply:
done